src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -3462,6 +3462,66 @@ static void Main()
             source.Append(source1);
             CompileAndVerify(source.ToString(), expectedOutput: "58430604");
         }
+
+        [Fact]
+        public void ConstantInterpolatedStringsSimple()
+        {
+            string source = @"
+class C
+{
+    void M()
+    {
+        const string S1 = $""Testing"";
+        const string S2 = $""{""Level 5""} {""Number 3""}"";
+        const string S3 = $""{$""{""Spinning Top""}""}"";
+        const string F1 = $""{S1}"";
+        const string F2 = $""{F1} the {S2}"";
+    }
+}";
+            var actual = ParseAndGetConstantFoldingSteps(source);
+
+            var expected =
+@"$""Testing"" --> Testing
+$""{""Level 5""} {""Number 3""}"" --> Level 5 Number 3
+$""{$""{""Spinning Top""}""}"" --> Spinning Top
+$""{""Spinning Top""}"" --> Spinning Top
+$""{S1}"" --> Testing
+$""{F1} the {S2}"" --> Testing the Level 5 Number 3";
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ConstantInterpolatedStringsError()
```

Comment:
We'll need more tests, but it's a good start. The rest can wait for test plan review.

Reply:
Think there are some we can add in advance of the test plan review: used in attributes, default parameter values, used across assembly boundaries, used as fields, etc ...  Also the always popular: lang ver checks. 

Those can be done in a separate PR but should get them done before a test plan review